webextension/js/popup.js

```diff
@@ -1,150 +1,335 @@
+  },
+
+  currentIdentity() {
+    if (!this._currentIdentity) {
+      throw("CurrentIdentity must be set before calling Logic.currentIdentity.");
+    }
+    return this._currentIdentity;
+  },
+};
+
+// P_ONBOARDING_1: First page for Onboarding.
+// ----------------------------------------------------------------------------
+
+Logic.registerPanel(P_ONBOARDING_1, {
+  panelSelector: ".onboarding-panel-1",
+
+  // This method is called when the object is registered.
+  initialize() {
+    // Let's move to the next panel.
+    document.querySelector("#onboarding-next-button").addEventListener("click", () => {
+      localStorage.setItem("onboarded1", true);
+      Logic.showPanel(P_ONBOARDING_2);
+    });
+  },
+
+  // This method is called when the panel is shown.
+  prepare() {
+    return Promise.resolve(null);
+  },
+});
+
+// P_ONBOARDING_2: Second page for Onboarding.
+// ----------------------------------------------------------------------------
+
+Logic.registerPanel(P_ONBOARDING_2, {
+  panelSelector: ".onboarding-panel-2",
+
+  // This method is called when the object is registered.
+  initialize() {
+    // Let's move to the containers list panel.
+    document.querySelector("#onboarding-done-button").addEventListener("click", () => {
+      localStorage.setItem("onboarded2", true);
+      Logic.showPanel(P_CONTAINERS_LIST);
+    });
+  },
+
+  // This method is called when the panel is shown.
+  prepare() {
+    return Promise.resolve(null);
+  },
+});
+
+// P_CONTAINERS_LIST: The list of containers. The main page.
+// ----------------------------------------------------------------------------
 
-  // Let"s retrieve the list of tabs.
-  browser.runtime.sendMessage({
-    method: "getTabs",
-    userContextId: identity.userContextId,
-  }).then(tabs => {
-    console.log('browser.runtime.sendMessage getTabs, tabs: ', tabs);
-    // For each one, let's create a new line.
+Logic.registerPanel(P_CONTAINERS_LIST, {
+  panelSelector: "#container-panel",
+
+  // This method is called when the object is registered.
+  initialize() {
+    document.querySelector("#container-add-link").addEventListener("click", () => {
+      Logic.showPanel(P_CONTAINER_EDIT, {});
+    });
+
+    document.querySelector("#edit-containers-link").addEventListener("click", () => {
+      Logic.showPanel(P_CONTAINERS_EDIT);
+    });
+
+    document.querySelector("#sort-containers-link").addEventListener("click", () => {
+      browser.runtime.sendMessage({
+        method: "sortTabs"
+      }).then(() => {
+        window.close();
+      });
+    });
+  },
+
+  // This method is called when the panel is shown.
+  prepare() {
     let fragment = document.createDocumentFragment();
-    for (const tab of tabs) {
+
+    Logic.identities().forEach(identity => {
+      log('identities.forEach');
       let tr = document.createElement("tr");
       fragment.appendChild(tr);
-      tr.classList.add("container-info-tab");
+      tr.classList.add("container-panel-row");
```

Comment:
Ref: https://github.com/mozilla/testpilot-containers/pull/82/files#r96049411

I think you can do this to add the two class names at once:
```js
tr.classList.add("container-panel-row", "clickable");
```